src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private final SimpleDateFormat formatter = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names here you can see how to create constant field